conda/recipes/libcudf/meta.yaml

```diff
@@ -31,12 +31,13 @@ requirements:
     - cudatoolkit {{ cuda_version }}.*
     - arrow-cpp 1.0.1.*
     - arrow-cpp-proc * cuda
-    - boost-cpp 1.72.0
+    - boost-cpp 1.72.0.*
```

Comment:
There’s no 4th version number for this release so would drop the `.*`

Reply:
I believe conda-build gives a warning without it, but I guess I should really update to `>=1.72.0,<1.73.0a0`

Reply:
The warning is probably about using a `conda_build_config.yaml` for this sort of thing, but I don't think we need to worry about that. This works just as well.

Would suggest the same change for `arrow-cpp` above.

Reply:
Are you saying that I should use the `>=1.72.0,<1.73.0a0` method or continue with using `1.72.0.*`?

Reply:
Personally would just drop the `.*` as Boost doesn't have a 4th version number. Also I don't think the other form works either. Boost doesn't have a stable ABI between patch releases. It's SONAME includes the patch version.

https://abi-laboratory.pro/index.php?view=timeline&l=boost

Reply:
If we just drop the `.*` then conda-build specifically warns about adding it back in... Is there an issue or doc or something to reference that says that this behavior is going to change or something of the like?

Reply:
Can you please share the warning?

Reply:
@jakirkham apologies you were correct. The warning was from packages that had `x.x.x` versions in which we were only specifying `x.x`